react/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "jsx": "react",
+        "module": "commonjs",
+        "target": "es3",
+        "strictNullChecks": true,
+        "outDir": "."
+    }
+}
```

Comment:
O, we does not need any configuration for jsx, just .tsx extension. I deleted the file.